cli/src/main/java/org/opentosca/toscana/cli/CliProperties.java

```diff
@@ -0,0 +1,99 @@
+package org.opentosca.toscana.cli;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+
+public class CliProperties {
+
+    private static String filePath;
+    private static String dataPath = null;
+    private static final String dataPathWin = "/AppData/toscana";
+    private static final String dataPathNix = "/.toscana";
+    private static final String cliProp = "/cli.properties";
+    private static Properties properties = new Properties();
+    private static File file;
+    private static String OS = null;
```

Comment:
This does not adhear to java code conventions replace this with `operatingSystem` (or simmilar)